content/docs/setup/kubernetes/quick-start.md

```diff
@@ -268,11 +268,19 @@ customize Istio to your needs.
 
     _**OR**_
 
-    b)  [Render Kubernetes manifest with Helm and deploy with kubectl](/docs/setup/kubernetes/helm-install/#option-1-install-with-helm-via-helm-template).
+    b) Install Istio and enable mutual TLS authentication globally between sidecars. This option is only recommended for new clusters, and all applications are installed with sidecars. Otherwise, we recommend to choose option a) and incrementally adopt Istio mutual TLS later on.
+
```

Comment:
grammar.
(c) Install Istio and enforce mutual TLS authentication by default. Use this option only on a fresh kubernetes cluster where newly deployed workloads are guaranteed to have Istio sidecars installed.

Reply:
done.